tests/stubs/extensionStorage.ts

```diff
@@ -111,3 +112,12 @@ export function getStubbedSecretStorage(sandbox: SinonSandbox): StubbedSecretSto
   sandbox.stub(storageUtils, "getSecretStorage").returns(stubbedSecretStorage);
   return stubbedSecretStorage;
 }
+
+/** Stub the singleton ResourceManager instance */
+export function getStubbedResourceManager(
+  sandbox: SinonSandbox,
+): SinonStubbedInstance<ResourceManager> {
+  const stubbed = sandbox.createStubInstance(ResourceManager);
+  sandbox.stub(ResourceManager, "getInstance").returns(stubbed);
+  return stubbed;
+}
```

Comment:
Nice addition!

Reply:
Was surprised we didn't have it already. I guess we've not been down in this layer lately since starting this pattern.